python/fledge/services/core/api/auth.py

```diff
@@ -625,25 +624,24 @@ async def update_password(request):
 
         # Remove OTT token for this user if there.
         __remove_ott_for_user(user_id)
-
     except ValueError as ex:
-        _logger.warning(str(ex))
+        _logger.error(str(ex))
         raise web.HTTPBadRequest(reason=str(ex))
     except User.DoesNotExist:
-        msg = "User with id:<{}> does not exist".format(int(user_id))
-        _logger.warning(msg)
+        msg = "User with id:<{}> does not exist.".format(int(user_id))
+        _logger.error(msg)
         raise web.HTTPNotFound(reason=msg)
     except User.PasswordAlreadyUsed:
-        msg = "The new password should be different from previous 3 used"
+        msg = "The new password should be different from previous 3 used."
         _logger.warning(msg)
         raise web.HTTPBadRequest(reason=msg)
     except Exception as exc:
         _logger.exception(str(exc))
         raise web.HTTPInternalServerError(reason=str(exc))
 
-    _logger.info("Password has been updated successfully for user id:<{}>".format(int(user_id)))
-
-    return web.json_response({'message': 'Password has been updated successfully for user id:<{}>'.format(int(user_id))})
+    msg = "Password has been updated successfully for user id:<{}>.".format(int(user_id))
```

Comment:
This is a good case of audio logging

Reply:
Audit log info is there and just after the line

https://github.com/fledge-iot/fledge/blob/67dbe4f9d00dfd52f49fcaf1224c40c0d9159387/python/fledge/services/core/api/auth.py#L643